handsontable/src/3rdparty/walkontable/src/renderer/rows.js

```diff
@@ -84,7 +84,7 @@ export default class RowsRenderer extends BaseRenderer {
         setAttribute(TR, [
           A11Y_ROW(),
           // `aria-rowindex` is incremented by both tbody and thead rows.
-          A11Y_ROWINDEX(sourceRowIndex + (this.table.rowUtils?.dataAccessObject?.columnHeaders.length ?? 0) + 1)
+          A11Y_ROWINDEX(sourceRowIndex + 1)
```

Comment:
This will not take into account the number of column header levels, and I think it should:

```
// `aria-rowindex` is incremented by both tbody and thead rows.
```

Reply:
@evanSe Please confirm. The specification says one thing and the [issue](https://github.com/handsontable/dev-handsontable/issues/1589) says another. I don't know what should be here.